kafka-transport/src/main/java/com/esri/geoevent/transport/kafka/KafkaInboundTransport.java

```diff
@@ -162,193 +101,114 @@ private synchronized void disconnect(String reason)
     }
   }
 
-  private synchronized void connect()
+  private void connect()
   {
-    disconnect("");
     setRunningState(RunningState.STARTING);
-    if (consumer == null)
-      consumer = new KafkaEventConsumer();
-    if (consumer.getStatusDetails().isEmpty()) // no errors reported while instantiating a consumer
+    shutdownFlag.set(false);
+    executorService = Executors.newFixedThreadPool(numThreads);
+    IntStream.range(0, numThreads).forEach(processingThread -> {
+      KafkaEventConsumer kafkaEventConsumer = new KafkaEventConsumer();
+      consumerList.add(kafkaEventConsumer);
+      executorService.submit(kafkaEventConsumer);
+    });
+  }
+
+  private void shutdownConsumer()
+  {
+    consumerList.forEach(eventConsumer -> {
+      eventConsumer.shutdown();
+    });
+    executorService.shutdown();
+    consumerList.clear();
+    try
     {
-      consumer.setConnected();
-      new Thread(this).start();
+      executorService.awaitTermination(5000, TimeUnit.MILLISECONDS);
     }
-    else
+    catch (InterruptedException e)
     {
-      setRunningState(RunningState.ERROR);
-      setErrorMessage(consumer.getStatusDetails());
+      e.printStackTrace();
```

Comment:
Log this (I don't think we care)?

```suggestion
      e.printStackTrace();
      // pass
```

Reply:
@eironside this is needed to log a message in the event something goes wrong during shutdown of the consumer(s)